jams/eval.py

```diff
@@ -54,10 +54,7 @@ def validate_annotation(ann, namespace):
         If `ann` fails schema validation
     '''
 
-    if not ann.search(namespace=namespace):
-        raise NamespaceError('Expected namespace="{:s}", found "{:s}"'
-                             .format(namespace, ann.namespace))
-
+    ann = convert(ann, namespace)
```

Comment:
Right. So let's say I have a ``pitch_midi`` annotation, and I validate it against the ``pitch_hz`` namespace, which would validate fine in this new scenario. So I then try to use this annotation with the transcription eval wrapper, which expects ``pitch_hz`` annotations, and it breaks.

Now of course in practice I'll be adding a conversion step to the transcription eval wrapper such that any annotation provided is converted into ``pitch_hz``, so this wouldn't actually happen. But could this type of scenario, where you validate against a namespace different to the one your annotation actually is (but convertible to) and consequently get errors further down the line somewhere?